src/Features/DynamicCubemaps.cpp

```diff
@@ -32,6 +33,73 @@ void DynamicCubemaps::DrawSettings()
 				ImGui::EndTable();
 			}
 		}
+
+		if (ImGui::TreeNodeEx("Dynamic Cubemap Creator", ImGuiTreeNodeFlags_DefaultOpen)) {
+			ImGui::Text("You must enable creator mode by adding the shader define CREATOR");
```

Comment:
It feels like we should auto do this for them or just not enable this menu unless that's set.